### PR TITLE
run tslint on travis

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,4 +15,5 @@ cache:
 
 script:
     - npm run test
+    - npm run tslint
```
Comment (tomaskikutis) [inline .travis.yml]: I think you should use `npm run lint` instead so eslint and other tools we might add run too.

Comment (petrjasek) [reply .travis.yml]: eventually I would like to, but it's failing on external libs and not sure atm how to fix that

Comment (tomaskikutis) [reply .travis.yml]: What does it say?

Comment (petrjasek) [reply .travis.yml]: eg.:
```
node_modules/@types/react/index.d.ts:2321:19 - error TS2320: Interface 'IntrinsicClassAttributes<T>' cannot simultaneously extend types 'ClassAttributes<T>' and 'ClassAttributes<T>'.
  Named property 'ref' of types 'ClassAttributes<T>' and 'ClassAttributes<T>' are not identical.

2321         interface IntrinsicClassAttributes<T> extends React.ClassAttributes<T> { 
```e

Comment (tomaskikutis) [reply .travis.yml]: Can't see what the source line of the error is. Does it originate from our code or something from node_modules?

Comment (petrjasek) [reply .travis.yml]: `node_modules/@types/react/index.d.ts:2321:19`

Comment (tomaskikutis) [reply .travis.yml]: That's strange, it looks like it's type checking the definitions themselves and not the code. Is there an easy way to get that error locally?

Comment (petrjasek) [reply .travis.yml]: I just run `tsc --noEmit` in client core repo

**Pull request merged**